tests/components/edl21/conftest.py

```diff
@@ -0,0 +1,15 @@
+"""Define test fixtures for EDL21."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.openuv.async_setup_entry", return_value=True
```

Comment:
I think you used the wrong domain here.
```suggestion
        "homeassistant.components.edl21.async_setup_entry", return_value=True
```